pages.es/linux/ping.md

```diff
@@ -0,0 +1,16 @@
+# ping
+
+> Comprueba la disponibilidad de un host.
+> Más información: <https://manpages.debian.org/bullseye/ping>.
+
+- Enviar paquetes ICMP a un host para comprobar que se encuentra disponible en la red.
+
+`ping {{host}}`
+
+- Enviar solo una cantidad específica de paquetes.
+
+`ping -c {{count}} {{host}}`
```

Comment:
```suggestion
`ping --count {{1..infinity}} {{host}}`
```

Mandatory fix: [long option must be used if it's portable](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).
Optional fix: `--count` value can be converted to range (can't deduce the highest value for `--count`, `infinity` used as a fallback.